src/datomic_toolbox/core.clj

```diff
@@ -229,3 +229,24 @@
         {:created-at (first tx-instants)
          :updated-at (last tx-instants)
          :timestamps tx-instants})))
+
+(defn retry-tx
```

Comment:
Ok, perhaps the name should be changed. Something like `swap!` maybe, and it takes a function of one argument (the current db) that returns datomic transaction data.

It seems also that the point of this function is not clear. I'll revise the PR description.